gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -284,6 +284,24 @@ final case class RandomLongRange(min: Long, max: Long) extends ElPart[Long] {
   def apply(session: Session): Validation[Long] = (min + ThreadLocalRandom.current().nextLong(length)).success
 }
 
+final case class RandomDoubleRange(min: Double, max: Double) extends ElPart[Double] {
+  require(min < max, s"'max'($max) should be greater than 'min'($min)")
+  private val minInclusive = min
+  private val maxExclusive = max
+  def apply(session: Session): Validation[Double] =
+    ThreadLocalRandom.current().nextDouble(minInclusive, maxExclusive).success
+}
+
+final case class RandomDoubleRangeDigits(min: Double, max: Double, fractionalDigits: Int) extends ElPart[Double] {
```

Comment:
Wondering if the only use case for this is not just a random amount of currency (where fractionalDigits is always 2 and we don't deal with huge amounts that might overflow).

Reply:
That's what I proposed for something of the sorts of currency or latitude/longitude, it can be time in seconds.nanoseconds as well. I don't think we will deal with large or small floating points. We can put a warning in the docs that near max Double or epsilon this will result in unexpected results.

Reply:
OK, you're right.

Reply:
Will do two changes:

1. Add details on docs, with warnings about floating points (large, epsilon)
2. Change the `digit` to `scale`